apps/nextjs-website/src/components/organisms/ChatbotLayout/ChatbotLayout.tsx

```diff
@@ -0,0 +1,99 @@
+import Chat from '@/components/molecules/Chat/Chat';
+import ChatButton from '@/components/atoms/ChatButton/ChatButton';
+import { Close } from '@mui/icons-material';
+import { Box, IconButton, Popover, Stack, Typography } from '@mui/material';
+import React from 'react';
+import { Query } from '@/lib/chatbot/queries';
+
+type ChatbotLayoutProps = {
+  queries: Query[];
+  onSendQuery: (query: string) => null;
+};
+
+const ChatbotLayout = ({ queries, onSendQuery }: ChatbotLayoutProps) => {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(
+    null
+  );
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+    return null;
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'chat-modal' : undefined;
+
+  return (
+    <Box
+      sx={{
+        position: 'fixed',
+        bottom: '2rem',
+        right: '2rem',
+        zIndex: 1000,
+      }}
+    >
+      <ChatButton
+        aria-describedby={id}
+        chatOpen={open}
+        onOpenChat={handleClick}
+        hasNewMessages={false}
+      />
+      <Popover
+        id={id}
+        open={open}
+        anchorEl={anchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right',
+        }}
+        transformOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right',
+        }}
+        disableScrollLock
+        slotProps={{
+          paper: {
+            sx: {
+              backgroundColor: 'transparent',
+              borderRadius: 3,
+              width: '40%',
+            },
+          },
+        }}
+      >
+        <Box
+          bgcolor={'black'}
+          padding={'0.75rem'}
+          borderRadius={3}
+          minWidth='40rem'
+        >
+          <Stack
+            direction='row'
+            justifyContent='space-between'
+            paddingBottom={'0.5rem'}
+            paddingX={'0.5rem'}
+          >
+            <Typography
+              variant='h5'
+              fontWeight='normal'
+              sx={{ color: 'white' }}
+            >
+              [Nome Chatbot]
+            </Typography>
+            <IconButton onClick={handleClose}>
+              <Close sx={{ color: 'white' }} />
```

Comment:
Maybe is better to use one of the following alternative here:
```suggestion
              <Close sx={{ color: palette.common.white }} />
```
Or:
```suggestion
              <Close sx={{ color: palette.primary.contrastText }} />
```

**NB** if u apply these remeber to manually add this line at the start of component:
 ```
  const { palette } = useTheme();
 ```